docs/clusters/alpine/allocations.md

```diff
@@ -99,16 +147,43 @@ Step 3: Look out for email messages from the CURC ticketing system (<rc-help@col
 is received, during the initial 
 review stages, and when the allocation is ready to use.
 
-### Renewing Your Allocation
+
+````
+
+````{tab-item} RMACC
+:sync: alpine-rmacc-inst
+
+**Ascent:**
+
+Step 1: Fill out the [RMACC Ascent Allocation 
+Request](https://forms.office.com/Pages/ResponsePage.aspx?id=G4vtPQ0HKUaC5MCwGfRgVxVuc407_5dMhLp4SuO1aoJUODlNNThXTTRUNklTQk02TlFKV1gxUUZTWCQlQCN0PWcu) form. You need to be 
+logged in to Office 365 with your CU Boulder account.
+
+Step 2: Look out for an email message from the CURC ticketing system (<rc-help@colorado.edu>) indicating when your allocation is ready to use.
+
+**Peak:**
+
+At this time, RMACC users are not eligible for Peak allocations. However, if you are an RMACC user with additional resource needs beyond the RMACC Ascent allocation, please contact us at (<rc-help@colorado.edu>).
+
+````
+
+````{tab-item} CSU & AMC
+:sync: alpine-other-inst
+
+**Allocations for CSU and AMC are managed by those institutions.** CSU and AMC users who are interested in learning about their institution's Alpine allocations processes should message (<rc-help@colorado.edu>) and (<hpcsupport@cuanschutz.edu>), respectively.
+
+````
+
+`````
+
+
+#### Renewing Your Allocation
```

Comment:
I am wondering if we should also put this into tabs. By having everything in tabs for this page, things will be a lot more clear.